guide/secure_secret_management.md

```diff
@@ -87,11 +88,11 @@ functions:
     image: functions/api-key-protected:latest
 ```
 
-Note that unlike the `envVars` in the first example, we do not provide the secret value, just a list of names: `"secrets": ["secret_api_key"]`. The secret value has already been securely stored in the Docker swarm.  One really great result of this type of configuration is that you can simplify your function code by always referencing the same secret name, no matter the environment, the only change is how the environments are configured.
+Note that unlike the `envVars` in the first example, we do not provide the secret value, just a list of names: `"secrets": ["secret_api_key"]`. The secret value has already been securely stored in the Docker swarm. One really great result of this type of configuration is that you can simplify your function code by always referencing the same secret name, no matter the environment, the only change is how the environments are configured.
```

Comment:
What changed on these lines? I can't see from the diff.

Reply:
Looks like the removal of an extra space

Reply:
<img width="115" alt="screen shot 2018-05-31 at 23 49 56" src="https://user-images.githubusercontent.com/83862/40812360-5f359afe-652d-11e8-9a51-5f384ef0331d.png">